lib/widgets/bubble_tab_indicator.dart

```diff
@@ -0,0 +1,113 @@
+library flutter_bubble_tab_indicator;
```

Comment:
if you have copied this from somewhere - add the license at the top. If you have not changed it, try to pull it through pubspec.yaml